changelog.d/12154.misc

```diff
@@ -0,0 +1 @@
+Avoid generating state groups for local out-of-band leaves.
```

Comment:
Is there anything to "clean-up" in the database from this?

Reply:
well in theory we might be able to go and delete the incorrectly created state groups. In practice, I'm not sure it's worthwhile.